ray-operator/DEVELOPMENT.md

```diff
@@ -39,6 +39,22 @@ export PATH="$GOROOT/bin:$PATH"
 
 Setting up workspace configuration is required because KubeRay contains multiple Go modules. See the [VS Code Go documentation](https://github.com/golang/vscode-go/blob/master/README.md#setting-up-your-workspace) for details.
 
+### IMPORTANT: Change your working directory to `ray-operator`
+
+All the following guidance require you to switch your working directory to the `ray-operator`.
+
+```bash
+cd ray-operator
+```
+
+### Cleanup local binaries, such as controller-gen and kustomize
+
+To keep consistent results of code generation and testing, you need to remove outdated binaries installed by the Makefile.
+
+```bash
+rm -rf bin
```

Comment:
Should we add a make target to clean build artifcacts locally?

Reply:
`make clean` is not a convention of kubebuilder projects but I think we can add it anyway.